language/move-prover/bytecode-to-boogie/src/bytecode_translator.rs

```diff
@@ -791,6 +769,26 @@ impl<'env> ModuleTranslator<'env> {
                 emitln!(self.writer, propagate_abort);
             }
             Ret(rets) => {
+                // Enforce invariants on mutable references not returned.
```

Comment:
Wouldn't we enforce this at the point they become dead? We don't need to fix this now, but just asking. If its the case, can you take the TODO we had here before (in reduced form) so it reminds us to do this?

Reply:
Sorry I just saw this! Here we are dealing with references that are still alive at return time. If a mutable reference is still alive at return time but is not one of the return values, then it should be considered dead and invariants should be enforced.